inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -663,6 +667,10 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
         // WA: ConvertPriorBox must be executed before the 1st ConstantFolding pass
         manager.register_pass<ngraph::pass::ConvertPriorBox>();
         manager.register_pass<ngraph::pass::CommonOptimizations>();
+        manager.register_pass<InsertTransposeBeforeMatmul>();
+        manager.register_pass<SwapInputMatMul>();
+        manager.register_pass<InsertTransposeAfterConvOrPool>();
+        manager.register_pass<ReorderActivationAndPooling>();
```

Comment:
Why do we need to have all these changes in one PR? It is pretty hard to review them.

Reply:
Actually all these changes are needed to enable 2 models, so they should be on the same branch to verify the final result.